website/pages/_app.tsx

```diff
@@ -17,6 +17,19 @@ function MyApp({ Component, pageProps }: AppProps) {
   const router = useRouter();
   const layout = router.pathname.split('/').filter(substr => !!substr)[0];
 
+  useEffect(() => {
+    document.querySelectorAll('a, button').forEach(e => {
+      // @ts-ignore
+      e.onclick = (e) => {
+        if(!!!localStorage.getItem('yaDoneBeenRolled')) {
```

Comment:
Can you get away with `!localStorage.getItem` instead of `!!!`?

Reply:
You could, I usually just do it when I'm using non-boolean values to check its truthy or falsiness. So `!!!` just indicates that it's a non-boolean var being used as a boolean.